.github/workflows/format-code.yml

```diff
@@ -1,45 +1,40 @@
-name: "Format code"
+name: 'Format code'
```

Comment:
Unrelated to this PR, but should we change this to "Format markdown files" to better match what it does?

Reply:
Yep, agreed. I've also renamed the file.